packages/core/src/browser/tree/tree-search.ts

```diff
@@ -18,7 +18,7 @@ import { inject, injectable, postConstruct } from 'inversify';
 import { Disposable, DisposableCollection } from '../../common/disposable';
 import { Event, Emitter } from '../../common/event';
 import { Tree, TreeNode } from './tree';
-import { TreeDecoration } from './tree-decorator';
+import { NodeDecoration } from '../node-decoration';
```

Comment:
Can we avoid breaking here? reexport types in `tree-decorator` from `node-decoration` and revert changes to clients of `tree-decorator`?

Reply:
Yep.